src/mlpack/tests/sfinae_test.cpp

```diff
@@ -134,4 +159,12 @@ BOOST_AUTO_TEST_CASE(HasMethodFormTest)
   static_assert(HasTemplatedM<B, MForm4, 1>::value, "value should be true");
 }
 
+BOOST_AUTO_TEST_CASE(HasMethodNameTest)
```

Comment:
Can you add a simple test description, as done for the other test cases?

Reply:
Added the description.